rocketpy/AeroSurface.py

```diff
@@ -174,10 +194,14 @@ def __init__(
         self._rocket_radius = rocket_radius
         self._base_radius = base_radius
         self._length = length
+        if bluffness is not None:
```

Comment:
This is duplicated with lines 330

Reply:
It is intended. It is done this way so that the bluffness is checked both when initializing the object and when changing the bluffness parameter (through the setter). We can put these lines of code into a function and call it instead, but I don't think that saves code lines

Also, I think the setters and getters structure in this class is a weird and makes the class more complex than it should be. Something to think about later

Reply:
> It is intended. It is done this way so that the bluffness is checked both when initializing the object and when changing the bluffness parameter (through the setter). We can put these lines of code into a function and call it instead, but I don't think that saves code lines

... it does not save code lines but is does avoid repeating code lines. But anyway I don't think this should be something to worry right now. In the future we could create some validators to check if a general input is being used within its valid range (mass, cds, radius, etc.). Right now your code is working and it is not killing anyone.

> Also, I think the setters and getters structure in this class is a weird and makes the class more complex than it should be. Something to think about later

Absolutely, you have no idea about how lost I'm with these setters in AeroSurface.py. 
This is definitely something to discuss later.